automation/test.sh

```diff
@@ -60,6 +60,9 @@ elif [[ $TARGET =~ cnao ]]; then
   export KUBEVIRT_PROVIDER=${TARGET/-cnao/}
   export KUBEVIRT_DEPLOY_CDI=false
 elif [[ $TARGET =~ sig-network ]]; then
+  if [[ $TARGET =~ sig-network-only ]]; then
```

Comment:
I think it wont work due to the `only` prefix.
You can remove this if altogether because this code already in the context of sig-network jobs term.

Reply:
I tested it with this 'only' and it worked.
Without it we can't control if we want to run the main suite or just the network suite.
Do you have other solution?

Reply:
https://github.com/kubevirt/kubevirt/pull/11689/commits/8bc9ea291be59abb40d2f76ff1e9dd24f17f840d#diff-ce25e369a9df9edc525425cc144fc8c40599fff3cc1e5058b64336d71af67162R76

this line strips `-sig-network*` so should be fine
else it wont even start for you because `KUBEVIRT_PROVIDER` must be a valid provider for cluster-up to start

Reply:
It wont work because sig-network jobs set TARGET to `k8s-1.29-sig-network` and it  doesnt  match the pattern `sig-network-only'.

> I tested it with this 'only' and it worked.
Without it we can't control if we want to run the main suite or just the network suite.
Do you have other solution?

Remove the new term altogether because it redundant, we already have dedicated lanes to run network test only.